modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/impl/WFSDataAccessFactory.java

```diff
@@ -400,6 +400,18 @@ public T lookUp(final Map params) throws IOException {
         parametersInfo[17] = GML_COMPLIANCE_LEVEL = new WFSFactoryParam<Integer>(name,
                 Integer.class, title, description, 0);
     }
+    
+    /**
+     * Optional {@code Integer} OCG GML compliance level. i.e. (simple feature) 0, 1 or 2
+     */
+    public static final WFSFactoryParam<Boolean> GML_COMPATIBLE_TYPENAMES;
+    static {
+        String name = "WFSDataStoreFactory:GML_COMPATIBLE_TYPENAMES";
+        String title = "GmlCompatibleTypeNames";
+        String description = "Use Gml Compatible TypeNames (replace : by _).";
+        parametersInfo[18] = GML_COMPATIBLE_TYPENAMES = new WFSFactoryParam<Boolean>(name,
+                Boolean.class, title, description, false);
```

Comment:
Is there any reason why this should be anything other than true?

I guess backwards compatibility for wfs-ng users, but since GML output would not of worked in this case previously...